frontend/src/components/core/Sidebar/Sidebar.tsx

```diff
@@ -70,8 +76,11 @@ class Sidebar extends PureComponent<SidebarProps, State> {
     this.mediumBreakpointPx = Sidebar.calculateMaxBreakpoint(
       props.theme.breakpoints.md
     )
+    this.minWidth = "336"
```

Comment:
I think we don't need this as a class variable, or? We could move it to a constant `const MIN_WIDTH="336"` or make it read-only, e.g.`static readonly minWidth = 336`

Reply:
Was the `336` supplied by design, we don't have it in any other format other than `px` or (e.g. `rem`)?